src/main/java/htsjdk/samtools/CRAMBAIIndexer.java

```diff
@@ -118,21 +126,21 @@ public CRAMBAIIndexer(final OutputStream output, final SAMFileHeader fileHeader)
      *
      * @param container container to be indexed
      */
-    public void processContainer(final Container container, final ValidationStringency validationStringency) {
+    void processContainer(final Container container, final ValidationStringency validationStringency) {
         if (container == null || container.isEOF()) {
             return;
         }
 
+        container.distributeIndexingParametersToSlices();

```

Comment:
Good catch: yes this can be removed.

There is a validation check later on, in `processAsSingleReferenceSlice()` because that is where all BAI code paths end up.

Since the indexing values all live in Slice (currently) I feel that's the appropriate place to validate them.  Maybe in the future that will change, but that's beyond the scope of this PR.